src/metabase/lib/metadata/calculation.cljc

```diff
@@ -424,6 +479,11 @@
   [_query _stage-number _x _options]
   [])
 
+;;; default impl is just the impl for [[expected-columns-method]]
+(defmethod visible-columns-method :default
+  [query stage-number x options]
+  (expected-columns-method query stage-number x options))
```

Comment:
Should we assert that `VisibleColumnsOptions` is a subtype of `ExpectedColumnsOptions`?

Reply:
Done